temboard

```diff
@@ -220,12 +221,21 @@ def main():
                           config.temboard['plugins_orm_engine'])
 
     # Task Manager
-    task_manager = TaskManager()
-    task_manager.set_logger(logger)
-    task_manager.task_list.filepath = os.path.join(config.temboard['home'],
-                                                   'task_list')
-    logger.info("Starting task manager.")
-    task_manager.start(parameters={'configpath': options.configfile})
+    tm = taskmanager.TaskManager(
+            task_path=os.path.join(config.temboard['home'], '.tm.task_list'),
+            address=os.path.join(config.temboard['home'], '.tm.socket')
```

Comment:
Question naïve, comment ça se passe pour la sécurité au niveau de la socket ? Qui peut écrire dans la socket ?

Reply:
Idem pour le fichier en fait.

Reply:
Every body can send data to the socket, we should restrict this by defining an `authkey` (default to '') shared only between `temboard` and `temboard-sched`.
For the backup file, there is an issue on `temboard-sched` project about restricting access rights to this file:  https://github.com/dalibo/temboard-sched/issues/13